opencxl/cxl/component/cxl_cache_dcoh.py

```diff
@@ -178,40 +184,42 @@ async def send_d2h_data(self, data: int, uqid: int):
         )
         await self._upstream_fifo.target_to_host.put(packet)
 
-    async def cxl_cache_readline(self, addr: int, cqid: Optional[int] = None) -> Future[int]:
-        loop = get_running_loop()
-        fut_data = loop.create_future()  # ugly solution
-
-        def _listen_check_set_fut(pckt: CxlCacheH2DRspPacket | CxlCacheH2DDataPacket):
-            # for now, ignore the 32B transfer case.
-            if isinstance(pckt, CxlCacheH2DDataPacket):
-                # received a data packet
-                # since we're ignoring the 32B transfer case,
-                # we can just assume this data packet contains everything we want
-                fut_data.set_result(pckt.data)
-                current_task().cancel()
-            if (
-                pckt.h2drsp_header.cache_opcode != CXL_CACHE_H2DRSP_OPCODE.GO
-                or pckt.h2drsp_header.rsp_data
-                in (CXL_CACHE_H2DRSP_CACHE_STATE.INVALID, CXL_CACHE_H2DRSP_CACHE_STATE.ERROR)
-            ):
-                current_task().cancel()  # terminate the listener and free the cqid
+    async def cxl_cache_readline(self, addr: int, cqid: Optional[int] = None) -> Awaitable[int]:
+        # TODO: Migrate the cqid logic to the state machine in the future
+        pass
```

Comment:
If this is not being used (presumably not), just comment out the whole function instead of putting a `pass`.